pkg/apisix/cluster.go

```diff
@@ -550,3 +557,27 @@ func readBody(r io.ReadCloser, url string) string {
 	}
 	return string(data)
 }
+
+// getSchema returns the schema of APISIX object.
+func (c *cluster) getSchema(ctx context.Context, url string) (string, error) {
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
```

Comment:
I am not sure if this url is right. Because I can not see the prefix contains `/schema/plugins/xxx`.
For example, if we want to get the schema of plugin `kafka-logger`. We can use this `http://127.0.0.1:9080/apisix/admin/schema/plugins/kafka-logger`.

Reply:
I found `/apisix/admin/schema/plugins/<plugin_name>` and `/apisix/admin/plugins/<plugin_name>` return the same result.